src/container/Table/ReminderTable.jsx

```diff
@@ -0,0 +1,109 @@
+import React from 'react';
+import AppBar from '@material-ui/core/AppBar';
+import PropTypes from 'prop-types';
+import Tab from '@material-ui/core/Tab';
+import Tabs from '@material-ui/core/Tabs';
+import TableMui from './TableMui';
+import { a11yProps } from './a11yProps';
+import { TabPanel } from './TabPanel';
+import { makeStyles } from '@material-ui/core/styles';
+import './style.scss';
+
+const useStyles = makeStyles((theme) => ({
+  table: {
+    minWidth: 100,
+    padding: '0px',
+  },
+  root: {
+    flexGrow: 1,
+    width: '100%',
+    padding: '0px',
+    backgroundColor: theme.palette.background.paper,
+  },
+}));
+
+const ReminderTable = (props) => {
+  // destructring props
+  const {
+    editRow,
+    deleteOldReminder,
+    allReminders,
+    pastReminders,
+    upcomingReminders,
+  } = props;
+
+  // Mui classes
+  const classes = useStyles();
+
+  // tab swithching value
+  const [value, setValue] = React.useState(0);
+  const handleChange = (event, newValue) => {
+    setValue(newValue);
+  };
+
+  return (
+    <div className={classes.root}>
+      <AppBar position='static' color='default'>
+        <Tabs
+          value={value}
+          onChange={handleChange}
+          indicatorColor='secondary'
+          textColor='secondary'
+          variant='scrollable'
+          scrollButtons='auto'
+          aria-label='scrollable auto tabs example'>
+          <Tab label='All Reminders' {...a11yProps(0)} />
+          <Tab label='Past Reminders' {...a11yProps(1)} />
+          <Tab label='UpComing Reminders' {...a11yProps(2)} />
+        </Tabs>
+      </AppBar>
+
+      {/* all reminders */}
+      <TabPanel value={value} index={0}>
```

Comment:
Can we have one json for all tab list and loop it.. can reuse the code

Reply:
currently I'm working on it.